docs/.eleventy.js

```diff
@@ -65,7 +65,7 @@ module.exports = function(eleventyConfig) {
     output +=   '<' + tag + ' class="grid--cell fl1 stacks-' + tag + '" id="'+ slug +'">';
     output +=     '<span class="pe-auto">' + text + '</span>';
     output +=   '</' + tag + '>';
-    output +=   '<a class="grid grid__center mbn6 s-btn s-btn__muted pe-auto" href="#'+ slug +'">';
+    output +=   '<a class="grid grid__center mbn6 s-btn s-btn__muted pe-auto" title="'+ text +'" href="#'+ slug +'">';
```

Comment:
This is a big improvement but I'm not a huge fan of `title` in this role because it acts as a description rather than a label.  I don't know what the right thing is.  Searched the web and spotted this: https://amberwilson.co.uk/blog/are-your-anchor-links-accessible/

Her fourth attempt was what I was going to recommend:

```
    output +=   '<a class="grid grid__center mbn6 s-btn s-btn__muted pe-auto" aria-label="link to this heading" href="#'+ slug +'" aria-describedby="' + slug + '">';
```

Her fifth attempt, and where her page is at now, is:

```
    output +=   '<a class="grid grid__center mbn6 s-btn s-btn__muted pe-auto" href="#'+ slug +'">';
    output +=   '    <span class="v-visible-sr">Section titled' + text + '</span>';
```